core/dbt/parser/manifest.py

```diff
@@ -440,20 +459,23 @@ def parse_project(
             self._perf_info.parsed_path_count + total_parsed_path_count
         )
 
-    # Loop through macros in the manifest and statically parse
-    # the 'macro_sql' to find depends_on.macros
-    def macro_depends_on(self):
+    # This should only be called after the macros have been loaded
+    def build_macro_resolver(self):
         internal_package_names = get_adapter_package_names(
             self.root_project.credentials.type
         )
-        macro_resolver = MacroResolver(
+        self.macro_resolver = MacroResolver(
```

Comment:
I don't think I completely understand this change. Why didn't we set `self.macro_resolver` before, and why are we doing it now?

Reply:
Because before this we only used the macro_resolver in one place in the ManifestLoader and now we're using it in two places.